aim2dat/strct/mixin.py

```diff
@@ -449,23 +449,56 @@ def delete_atoms(
     @manipulates_structure
     def scale_unit_cell(
         self,
-        scaling_factor: float = 1.0,
-        change_label: bool = False,
+        scaling_factors: Union[float, List[float]] = None,
+        pressure: float = None,
+        bulk_modulus: float = None,
+        change_label: bool = True,
     ) -> Union["Structure", "StructureCollection"]:
```

Comment:
```suggestion
    ) -> Union[Structure, StructureCollection]:
```

This should be type: `Structure` or `StructureCollection` if you use `""` this is considered as a `str` in the list.

Reply:
Exactly, everything is fine from my side. Moreover, I only added the links in case anyone is curious. 

I just created an issue to make things consistent.